src/app/components/user/user-from/user-form.component.html

```diff
@@ -49,7 +49,7 @@ <h3 class="mb-5 px-5">{{title}}</h3>
         <input #lastname="ngModel" type="text" class="form-control" id="lastname" 
           name="lastname" 
           placeholder="Hernández"
-          [(ngModel)]="user.lastname" 
+          [(ngModel)]="user.last_name" 
```

Comment:
Please use camel case to variable names.

Reply:
Named as backend variable